app/models/concerns/nested_ancestry_common.rb

```diff
@@ -85,7 +85,7 @@ def set_title
 
   def set_other_titles
     if saved_change_to_name? || saved_change_to_ancestry?
-      self.class.where.not(ancestry: nil).find_each do |obj|
+      self.class.where('ancestry IS NOT NULL').find_each do |obj|
```

Comment:
would this work?
```suggestion
      self.class.where.not('ancestry' => nil).find_each do |obj|
```

Reply:
i think it would be the same. i'm still trying to figure out which of these changes is breaking katello, once i do i'll see if i can easily fix it or i'll just leave that one out